ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS.yml

```diff
@@ -70,7 +70,17 @@ tenants:
           121:
             name: Tenant_A_WEBZone_2
             tags: ['web']
-            profile: GENERIC_FULL
+            nodes:
```

Comment:
Please don't remove stuff from existing Vlans. It will remove test coverage of other features. Add a new Vlan to test your combination unless it can be done without impact to the existing tests